python/opae.admin/opae/admin/fpga.py

```diff
@@ -207,7 +207,7 @@ def i2c_bus(self):
     @property
     def spi_bus(self):
         if os.path.basename(self.sysfs_path).startswith('dfl'):
-            return self.find_one('dfl-fme.*.*/spi*/spi_master/spi*/spi*')
+            return self.find_one('dfl-fme.*.*/subdev_spi_altera.*.auto/spi_master/spi*/spi*')
```

Comment:
This change did work.  You could also use *spi* instead of subdev_spi_altera.*.auto

Reply:
As @fpgamatt says, you can probably use:
```Python
            return self.find_one('dfl-fme.*.*/*spi*/spi_master/spi*/spi*')
```